src/test/nft.t.sol

```diff
@@ -16,274 +16,96 @@
 pragma solidity >=0.4.23;
 pragma experimental ABIEncoderV2;
 
-import "ds-test/test.sol";
 import "../nft.sol";
+import "../../lib/ds-test/src/test.sol";
 
-contract AnchorMock {
-    bytes32 documentRoot;
-    uint32  blockNumber;
+contract MockAssetManager {
+    mapping (bytes32 => uint8) public assets;
 
-    function file(bytes32 documentRoot_, uint32 blockNumber_) public {
-        documentRoot = documentRoot_;
-        blockNumber = blockNumber_;
+    function store(bytes32 asset) public {
```

Comment:
We use `file` throughout our codebase, please keep this convention unless it's implementing an specific interface.

Reply:
What does `file` mean here?
I'm not familiar with solidity convictions we are following, is there any document I should take a look at @lucasvo ?